paasta_tools/metrics/metastatus_lib.py

```diff
@@ -32,9 +32,10 @@
 from paasta_tools.utils import PaastaColors
 from paasta_tools.utils import print_with_indent
 
-
+class ResourceInfo(namedtuple('ResourceInfo', ['cpus', 'mem', 'disk', 'gpus'])):
```

Comment:
I can't because there are parts of paasta that used namedtuple specific functions.